compiler/rustc_lint/src/early/diagnostics.rs

```diff
@@ -367,5 +367,8 @@ pub fn decorate_attribute_lint(
             suggestion: lints::UnsafeAttrOutsideUnsafeSuggestion { left, right },
         }
         .decorate_lint(diag),
+        &AttributeLintKind::EmptyCfgPredictate { predicate_span, predicate, lit } => {
```

Comment:
same for this, don't think this is emitted through here anymore? Since it's now an early lint, not an attribute lint anymore.